### PR TITLE
[HDInsight] Migrate adal graph sdk to msgraph sdk in HDInsight module

### PR DESCRIPTION
**Pull request by aim-for-better**
**Related command**

Only two completers are impacted.

**Description**

This PR migrate ADAL graph SDK to MS graph SDK.

There is not customer impact

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
HDInsight

**Comment by aim-for-better** [comment]
Hi team, I find that I can not change the PR title, I added one space to fix the format issue, but after I refersh the page, there is not space.

**Comment by jiasli** [comment]
This is happening again - "Azure.azure-cli Full Test" is not triggered. @wangzelin007

File: src/azure-cli/azure/cli/command_modules/hdinsight/_client_factory.py [hunk 1]
```diff
@@ -25,15 +25,8 @@ def cf_network(cli_ctx, aux_subscriptions=None, **_):
...
-    from azure.cli.core.commands.client_factory import configure_common_settings
-    from azure.graphrbac import GraphRbacManagementClient
-    profile = Profile(cli_ctx=cli_ctx)
-    cred, _, tenant_id = profile.get_login_credentials(
-        resource=cli_ctx.cloud.endpoints.active_directory_graph_resource_id)
-    client = GraphRbacManagementClient(cred, tenant_id,
-                                       base_url=cli_ctx.cloud.endpoints.active_directory_graph_resource_id)
-    configure_common_settings(cli_ctx, client)
+    from azure.cli.command_modules.role import graph_client_factory
+    client = graph_client_factory(cli_ctx)
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/hdinsight/_client_factory.py:29]: I am glad to see the new `azure.cli.command_modules.role.graph_client_factory` interface significantly reduces the effort to build a Graph client. 😊

**Review by jiasli — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/hdinsight/_client_factory.py (hunk 1 above)
Comment (aim-for-better) [reply src/azure-cli/azure/cli/command_modules/hdinsight/_client_factory.py:29]: > I am glad to see the new `azure.cli.command_modules.role.graph_client_factory` interface significantly reduces the effort to build a Graph client. 😊

learnt this from key vault PR 😁

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**